lib/FileUploadDataSource.ts

```diff
@@ -112,22 +174,10 @@ export default class FileUploadDataSource extends RemoteGraphQLDataSource {
 
     // This must come before the file contents append bellow
     form.append('map', JSON.stringify(fileMap));
-
-    resolvedFiles.forEach(
-      ({ createReadStream, filename, mimetype: contentType }, i: number) => {
-        form.append(i.toString(), createReadStream(), {
-          contentType,
-          filename,
-          /*
-              Set knownLength to NaN so node-fetch does not set the
-              Content-Length header and properly set the enconding
-              to chunked.
-              https://github.com/form-data/form-data/pull/397#issuecomment-471976669
-          */
-          knownLength: Number.NaN,
-        });
-      },
-    );
+    const handler = this.useChunkedTransfer
```

Comment:
I'd do this in the constructor, instead of `this.useChunkedTransfer`, I'd set the function to the instance, as it will not change.